models/articles.models.js

```diff
@@ -62,9 +62,22 @@ const selectArticleComments = (id) => {
           });
     });
 };
+
+const insertComment = (id, username, body) => {
+  return db
+    .query(
+      `INSERT INTO comments (body, votes, author, article_id) VALUES ($3, 0, $2, $1 ) RETURNING *`,
+      [id, username, body]
+    )
```

Comment:
Your votes should default to 0 so shouldn't be needed here. I'd also advocate for sequential ordering here to avoid confusion for anyone else reading your code.